src/browser/setupWorker/start/utils/createMessageChannel.ts

```diff
@@ -30,8 +30,9 @@ export class WorkerChannel {
     this.port.postMessage(
       { type: event, data },
       {
-        // @ts-ignore ReadableStream can be transferred
         // but TypeScript doesn't acknowledge that.
+        // eslint-disable-next-line @typescript-eslint/prefer-ts-expect-error
+        // @ts-ignore ReadableStream can be transferred, but only in versions 4.8 and lower
```

Comment:
this is the line I initially wanted to fix, but we can't, because it doesn't work in 4.8.

If we drop typescript 4.8, it works fine though